packages/interactivity/docs/api-reference.md

```diff
@@ -659,7 +659,7 @@ It would generate the following output:
 </ul>
 ```
 
-The prop that holds the item in the context can be changed by passing a suffix to the directive name. In the following example, we change the default prop `item` to `greeting`.
+The prop that holds the item in the context can be changed by passing a suffix to the directive name. In the following example, the default prop changes from `item` to `greeting`.
 
 ```html
 <ul data-wp-context='{ "list": [ "hello", "hola", "olá" ] }'>
```

Comment:
I'm still a little confised when you would use `wp_interactivity_state` over a raw `data-wp-context`